lib/runtime/wasmer/interfaces.go

```diff
@@ -58,6 +60,7 @@ type BasicNetwork interface {
 // TransactionState is the interface for the transaction state.
 type TransactionState interface {
 	AddToPool(vt *transaction.ValidTransaction) common.Hash
+	PopWithTimer(timerCh <-chan time.Time) (transaction *transaction.ValidTransaction)
```

Comment:
Why is this addition needed? It compiles fine with it commented 🤔

Reply:
good catch, not sure why that ended up being added, thank ya